spec/features/applications/show_spec.rb

```diff
@@ -175,4 +175,37 @@
     end
   end
 
+  # When I visit an application show page
+  # And I search for Pets by name
+  # Then I see any pet whose name PARTIALLY matches my search
+  # For example, if I search for "fluff", my search would match pets with names "fluffy", "fluff", and "mr. fluff"
+
+  describe "User Story 8" do
+    describe "Partial matches" do
+      it 'can see any pet whos name partially matches the search' do
+        @pet_1 = @application_1.pets.create!(
```

Comment:
This will create Pepper as a child of @application_1, so Pepper shows up on our Applications Show page under the "Pets applying for" section.